packages/gravity-ui-web/src/ui-lib/sass/05-components/03-organisms/00-page-structure/_01-page-footer.scss

```diff
@@ -1,35 +1,64 @@
-body > footer {
-  @include typi('small-text');
+.grav-c-footer {
+  $breakpoint-medium: gravy-breakpoint(medium);
+
   margin-top: vr(3);
   padding: $grav-sp-l 0;
+  background: #424242;
+  border-bottom: 2px solid #00b49d;
+  color: #9b9b9b;
+  font-size: 16px;
 
-  > * {
-    @include grav-l-container;
-    padding-bottom: $grav-sp-s;
-    text-align: center;
+  &__logo {
+    max-width: 100px;
+    margin-bottom: $grav-sp-m;
 
-    &:last-child {
-      padding: 0;
+    @media all and (max-width: $breakpoint-medium) {
+      display: block;
+      margin: 0 auto;
     }
   }
 
-  .grav-c-logo {
-    max-width: 7rem;
-    height: 1rem;
-    border-bottom: 1px solid currentColor;
-  }
+  &__lists {
+    display: flex;
+    margin: 0 0 $grav-sp-m;
 
-  .grav-c-icon {
-    width: 24px; // To ensure icon renders at the correct size in IE
-    height: 24px;
+    @media all and (max-width: $breakpoint-medium) {
+      flex-direction: column;
+      text-align: center;
+    }
   }
 
-  ul {
-    justify-content: center;
+  &__lists-entry {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-decoration: none;
+
+    @media all and (min-width: $breakpoint-medium) {
+      &:first-of-type {
+        padding-right: 100px;
+      }
+    }
+
+    li {
+      margin: 0;
+      padding: 0;
+    }
+
+    a[href]:link {
```

Comment:
Hmm.. I'm tempted to remove this entire `a .. { ... }` block.

- The link colors will already be set by the color scheme, so we should not be resetting them explicitly.
- The font-weight should just be inherited, since it's the same as the default for the page anyway
- That leaves removing the underline... which, TBH, I think is questionable design choice (if text links are underlined anywhere else, why shouldn't that be true inside the footer too?)

Yeah, let's just strip this away for now. Once it's all up and running together with the finalised color schemes and correct typography (both of which will be handled by other PRs) we can review it. If Paddy or others feel strongly about removing underlines of links in a footer, then we can (grdugingly) add that rule back in :-P